### PR TITLE
Add poster video link

### PR DESCRIPTION
**Pull request by raynerljm**
- [x] Add `posterUrl` and `videoUrl` attribute to project type
- [x] Add rendering of poster URL and video URL to project card
- [x] Add the URLs to the `/projects/:projectId` page for public viewing
- [x] Add the option to edit the URLs on the edit project page

Depends on BE PR to:
1. Add posterUrl and videoUrl to schema (optional attribute)
2. Update the respective CRUD endpoints

### CODE REVIEW

File: pages/projects/[projectId]/index.tsx [hunk 1]
```diff
@@ -66,7 +66,7 @@ const ProjectDetails: NextPage = () => {
         <Stack direction="column" alignItems="center">
           <Box
             component="img"
-            src={"https://nusskylab-dev.comp.nus.edu.sg/posters/2021/2680.jpg"}
+            src={project?.posterUrl}
             alt={`${project?.name} Project`}
```
Comment (myc37) [inline pages/projects/[projectId]/index.tsx:68]: can consider having a fallback here

**Review by myc37 — APPROVED** [review]
lgtm, think it would be good to have a fallback poster

File: pages/projects/[projectId]/index.tsx (hunk 1 above)
Comment (raynerljm) [reply pages/projects/[projectId]/index.tsx:68]: Added

**Review by myc37 — APPROVED** [review]
lgtm!

**Pull request merged**